django_request_cache/request.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+"""
+Vendorized from @beachmachine/django-userforeignkey, BSD Licensed
+https://github.com/beachmachine/django-userforeignkey/blob/master/django_userforeignkey/request.py
+"""
+import logging
+from threading import local
```

Comment:
Isn't this supposed to be `asgiref.local` now?

Reply:
I've just spotted you said you didn't do the async part yet to make sure vendorizing works, sorry.